packages/rs-drive/src/prove/mod.rs

```diff
@@ -1,3 +1,4 @@
 mod prove_elements;
 mod prove_multiple_state_transition_results;
+#[cfg(feature = "server")]
```

Comment:
@QuantumExplorer this is just monkey-patched, please fix correctly in your branch